invenio_records_resources/services/records/facets/facets.py

```diff
@@ -266,7 +275,15 @@ def __init__(self, field=None, label=None, value_labels=None, **kwargs):
         kwargs["field"] = self.field(field)
         super().__init__(label, value_labels, **kwargs)
 
-    @classmethod
-    def field(cls, field):
-        """Format field with `custom_fields` prefix."""
-        return f"custom_fields.{field}"
+
+class CFNestedTermsFacet(CFFacetMixin, NestedTermsFacet):
+    """Nested Terms facet for custom fields.
+
+    Works exactly as NestedTermsFacet except that prepends the prefix `custom_fields`
+    in the field definition.
+    """
+
+    def __init__(self, field=None, label=None, value_labels=None, **kwargs):
```

Comment:
question: can we also move the `__init__` to the constructor and play with MRO so both `CFTermsFacet` and `CFNestedFacet` are just a one liner?